src/lib/stores/filtered_items_store.ts

```diff
@@ -0,0 +1,30 @@
+import type { Filters } from 'src/types/filters.type';
+import type { Item } from 'src/types/item.type';
+import { writable } from 'svelte/store';
+
+let gItems: Item[];
+let gFilter: Filters;
+let itemDB: Item[];
```

Comment:
But in all cases, I'ld remove these three. We don't like global variables. They make state changes with the stores a lot more confusing.